doppelganger/bayesnets.py

```diff
@@ -96,7 +97,7 @@ def to_json(self):
         blob['type_to_network'] = {
             type_: json.loads(network.to_json()) for type_, network in self.type_to_network.items()
         }
-        return json.dumps(blob, indent=4, sort_keys=True)
+        return str(json.dumps(blob, indent=4, sort_keys=True, ensure_ascii=False))
```

Comment:
No worries! No need to make a new pull request. You can change the file, commit the change, then push it to this same branch.  Github will update this pull request automatically.